src/utils/profileContents.js

```diff
@@ -0,0 +1,8 @@
+const profileContents = userData => `{
```

Comment:
This could be refactored to something like fileContents.js

```
const profileContents = userData =>   
  JSON.stringify({
    avatar: userData.avatar,
    bio: userData.bio,
    name: userData.name,
    follows: userData.follows
  });

 export default profileContents;
```